### PR TITLE
Removed all usage of implicit unwrapped optionals on CameraManager

### PR DESCRIPTION
**Pull request by elikohen**

### CODE REVIEW

File: camera/CameraManager.swift [hunk 1]
```diff
@@ -84,10 +84,9 @@ open class CameraManager: NSObject, AVCaptureFileOutputRecordingDelegate, UIGest
 
     /// The Bool property to determine if current device has front camera.
     open var hasFrontCamera: Bool = {
-        let devices = AVCaptureDevice.devices(withMediaType: AVMediaTypeVideo)
-        for  device in devices!  {
-            let captureDevice = device as! AVCaptureDevice
-            if (captureDevice.position == .front) {
+        guard let devices = AVCaptureDevice.devices(withMediaType: AVMediaTypeVideo) else { return false }
+        for device in devices  {
+            if let captureDevice = device as? AVCaptureDevice, captureDevice.position == .front {
```
Comment (ahernandezlopez) [inline camera/CameraManager.swift]: this looks like can be replaced with a filter, right? :)

File: camera/CameraManager.swift [hunk 2]
```diff
@@ -96,10 +95,9 @@ open class CameraManager: NSObject, AVCaptureFileOutputRecordingDelegate, UIGest
 
     /// The Bool property to determine if current device has flash.
     open var hasFlash: Bool = {
-        let devices = AVCaptureDevice.devices(withMediaType: AVMediaTypeVideo)
-        for  device in devices!  {
-            let captureDevice = device as! AVCaptureDevice
-            if (captureDevice.position == .back) {
+        guard let devices = AVCaptureDevice.devices(withMediaType: AVMediaTypeVideo) else { return false }
+        for device in devices  {
+            if let captureDevice = device as? AVCaptureDevice, captureDevice.position == .back {
```
Comment (ahernandezlopez) [inline camera/CameraManager.swift]: same here, can be replaced w a filter?

File: camera/CameraManager.swift (hunk 1 above)
Comment (elikohen) [inline camera/CameraManager.swift]: yep, i didn't want to put everything pretty (as i could end up re-writing almost everything), but i can change this case

File: camera/CameraManager.swift [hunk 3]
```diff
@@ -962,7 +950,7 @@ open class CameraManager: NSObject, AVCaptureFileOutputRecordingDelegate, UIGest
         }
         if (movieOutput == nil) {
             movieOutput = AVCaptureMovieFileOutput()
-            movieOutput!.movieFragmentInterval = kCMTimeInvalid
+            movieOutput?.movieFragmentInterval = kCMTimeInvalid
         }
         if library == nil {
             library = PHPhotoLibrary.shared()
```
Comment (ahernandezlopez) [inline camera/CameraManager.swift]: scared of reading PHP in Swift code :rofl:

**Pull request merged**